Sources/App/Views/PublicPage.swift

```diff
@@ -42,7 +42,7 @@ class PublicPage {
             ),
             .link(
                 .rel(.stylesheet),
-                .href("/stylesheets/main.min.css?\(resourceReloadQueryString())")
+                .href("stylesheets/main.min.css?\(resourceReloadQueryString())")
```

Comment:
I think we should use `SiteURL` here in a similar way we do for images and that should give us the control to DI a variant for testing.

I experimented with that briefly when I was trying to explore if absolute file paths would help render the css. If we use `SiteURL` we can make this path be anything we want in the tests.

Reply:
This seems like a sensible way to address Dave's concerns too!